src/stp/new_tactics/Halt.cpp

```diff
@@ -0,0 +1,37 @@
+//
+// Created by jessevw on 24.03.20.
+//
+
+#include "include/roboteam_ai/stp/new_tactics/Halt.h"
+#include "stp/new_skills/GoToPos.h"
+#include "stp/new_skills/Rotate.h"
+
+namespace rtt::ai::stp::tactic {
+    Halt::Halt() {
+        skills = collections::state_machine<Skill, Status, StpInfo>{skill::GoToPos(), skill::Rotate()};
```

Comment:
Do we need GoToPos?

Reply:
Now that I think of it, we do. Because otherwise the robot keeps its inertia and can just bump into anything, whereas if we try to make the robot hold its own position it will slow down

Reply:
I don't think that is necessary. When you don't send a velocity to the robot, which is the case when you Rotate, the robot itself will actively adjust its motors to decelerate.

Reply:
Good point, updated it